zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/ZiplineApis.kt

```diff
@@ -33,132 +35,132 @@ internal class ZiplineApis(
 ) {
   private val packageFqName = FqName("app.cash.zipline")
   private val bridgeFqName = FqName("app.cash.zipline.internal.bridge")
+  private val collectionsFqName = FqName("kotlin.collections")
   private val serializationFqName = FqName("kotlinx.serialization")
-  val contextualFqName = serializationFqName.child("Contextual")
+  val contextualClassId = serializationFqName.classId("Contextual")
   private val serializationModulesFqName = FqName("kotlinx.serialization.modules")
-  private val serializersModuleFqName = serializationModulesFqName.child("SerializersModule")
-  private val ziplineFqName = packageFqName.child("Zipline")
-  private val outboundServiceFqName = bridgeFqName.child("OutboundService")
-  val ziplineScopedFqName = packageFqName.child("ZiplineScoped")
-  val ziplineServiceFqName = packageFqName.child("ZiplineService")
-  private val ziplineServiceSerializerFunctionFqName =
-    packageFqName.child("ziplineServiceSerializer")
-  private val ziplineServiceAdapterFunctionFqName = bridgeFqName.child("ziplineServiceAdapter")
-  private val ziplineServiceAdapterFqName = bridgeFqName.child("ZiplineServiceAdapter")
-  private val endpointFqName = bridgeFqName.child("Endpoint")
-  private val suspendCallbackFqName = bridgeFqName.child("SuspendCallback")
-  val flowFqName = FqName("kotlinx.coroutines.flow").child("Flow")
-  private val collectionsFqName = FqName("kotlin.collections")
-  private val listFqName = collectionsFqName.child("List")
+  private val serializersModuleClassId = serializationModulesFqName.classId("SerializersModule")
+  private val ziplineClassId = packageFqName.classId("Zipline")
+  private val outboundServiceClassId = bridgeFqName.classId("OutboundService")
+  val ziplineScopedClassId = packageFqName.classId("ZiplineScoped")
+  val ziplineServiceClassId = packageFqName.classId("ZiplineService")
+  private val ziplineServiceSerializerFunctionCallableId = packageFqName.callableId("ziplineServiceSerializer")
+  private val ziplineServiceAdapterFunctionCallableId = bridgeFqName.callableId("ziplineServiceAdapter")
+  private val ziplineServiceAdapterClassId = bridgeFqName.classId("ZiplineServiceAdapter")
+  private val endpointClassId = bridgeFqName.classId("Endpoint")
+  private val suspendCallbackClassId = bridgeFqName.classId("SuspendCallback")
+  val flowClassId = FqName("kotlinx.coroutines.flow").classId("Flow")
+  private val anyClassId = FqName("kotlin").classId("Any")
```

Comment:
```suggestion
  private val anyClassId = StandardClassIds.Any
```

Reply:
Similar Ids exist for List, Map and also the base Collections Package

Reply:
Good callout, thanks!